### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by SashaViun**
[DEMO LINK](https://SashaViun.github.io/react_sum/)

### CODE REVIEW

File: src/components/Sum.jsx [hunk 1]
```diff
@@ -1 +1,17 @@
 // export function Sum() {}
```
Comment (NVyrsta) [inline src/components/Sum.jsx:1]: don't forget to delete redundant comments

**Review by NVyrsta — APPROVED** [review]
Good job :0)

File: src/components/Sum.jsx (hunk 1 above)
Comment (alexandra-protyanova) [reply src/components/Sum.jsx:1]: delete it

**Review by alexandra-protyanova — APPROVED** [review]
Well done 👍

**Pull request closed without merging**